code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -0,0 +1,753 @@
+GLOBAL_LIST_EMPTY(lifts)
+
+/obj/structure/industrial_lift
+	name = "lift platform"
+	desc = "A lightweight lift platform. It moves up and down."
+	icon = 'icons/obj/smooth_structures/catwalk.dmi'
+	icon_state = "catwalk-0"
+	base_icon_state = "catwalk"
+	density = FALSE
+	anchored = TRUE
+	armor = list(MELEE = 50, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 80, ACID = 50)
+	max_integrity = 50
+	layer = LATTICE_LAYER //under pipes
+	plane = FLOOR_PLANE
+	smoothing_flags = SMOOTH_BITMASK
+	smoothing_groups = list(SMOOTH_GROUP_INDUSTRIAL_LIFT)
+	canSmoothWith = list(SMOOTH_GROUP_INDUSTRIAL_LIFT)
+	obj_flags = CAN_BE_HIT | BLOCK_Z_OUT_DOWN
+	appearance_flags = PIXEL_SCALE|KEEP_TOGETHER //no TILE_BOUND since we're potentially multitile
+
+	var/id = null //ONLY SET THIS TO ONE OF THE LIFT'S PARTS. THEY'RE CONNECTED! ONLY ONE NEEDS THE SIGNAL!
```

Comment:
autodoc this

Reply:
this var doesnt appear to be used anymore so ill just remove it